### PR TITLE
Firefight misc fixes and improvements

### PR DESCRIPTION
**Pull request by caelaislinn**
See commit log. Intending to address #2435 
:cl:
bugfix: NPCs in Firefight/Crusade/Stranded should no longer get lost on the way to the base.
bugfix: Fix the equip bug with jackal shields and gives them more visual feedback
experiment: Reduces bullet attack log spam
/:cl:

### CODE REVIEW

File: code/modules/halo/covenant/jobs/jiralhanae.dm [hunk 1]
```diff
@@ -4,27 +4,24 @@
 	total_positions = 1
 	spawn_positions = 1
 	outfit_type = /decl/hierarchy/outfit/jiralhanae/covenant/captain
-	access = list(240,250)
```
Comment (bloxgate) [inline code/modules/halo/covenant/jobs/jiralhanae.dm:7]: Is this removal intended?

File: code/modules/halo/covenant/jobs/sangheili.dm [hunk 2]
```diff
@@ -45,8 +46,8 @@
 
 /datum/job/covenant/sangheili_minor
 	title = "Sangheili Minor"
-	total_positions = 4
-	spawn_positions = 4
+	total_positions = -1
```
Comment (bloxgate) [inline code/modules/halo/covenant/jobs/sangheili.dm:49]: Is there a reason this isn't a 0 rather than -1?

**Review by bloxgate — COMMENTED** [review]
Looks good on a quick skim, just a couple questions

File: code/modules/halo/covenant/jobs/sangheili.dm (hunk 2 above)
Comment (caelaislinn) [reply code/modules/halo/covenant/jobs/sangheili.dm:49]: -1 means the job is uncapped

File: code/modules/halo/covenant/jobs/jiralhanae.dm (hunk 1 above)
Comment (caelaislinn) [reply code/modules/halo/covenant/jobs/jiralhanae.dm:7]: Yeah I've set the access in its parent `/datum/job/covenant`

**Review by bloxgate — APPROVED** [review]
